gptqmodel/models/base.py

```diff
@@ -590,8 +591,13 @@ def forward(self, *args, **kwargs):
 
     def generate(self, **kwargs):
         """shortcut for model.generate"""
-        with torch.inference_mode(), torch.amp.autocast(device_type=self.device.type):
-            return self.model.generate(**kwargs)
+        load_format = kwargs.pop('load_format', None)
```

Comment:
Remove `load_format`, there is no need to pass this. Set model._backend to `backend` and use it.

Reply:
Check later comment to see if we can infer type of model.config to vllm_config.